foundry/applications/formApplications/baseEntitySheets/journalSheet.d.ts

```diff
@@ -1,23 +1,73 @@
 /**
  * The JournalEntry Configuration Sheet
- *
- * @param entity - The JournalEntry instance which is being edited
- * @param options - Application options
  */
-declare class JournalSheet extends BaseEntitySheet {
+declare class JournalSheet<
+  D extends object = BaseEntitySheet.Data<JournalEntry>,
+  O extends JournalEntry = D extends BaseEntitySheet.Data<infer T> ? T : JournalEntry
+> extends BaseEntitySheet<D, O> {
   /**
-   * Provide a unique CSS ID for Entity Sheets
+   * @param entity - The JournalEntry instance which is being edited
+   * @param options - JournalSheet options
    */
+  constructor(entity: O, options?: JournalSheet.Options);
+
+  /** @override */
+  static get defaultOptions(): JournalSheet.Options;
+
+  /** @override */
   get id(): string;
 
+  /** @override */
+  get template(): string;
+
+  /** @override */
+  get title(): string;
+
+  protected _sheetMode: JournalSheet.SheetMode | null;
```

Comment:
Please move it directly below the constructor. If possible, we lick to stick to the order that things appear in foundry.js and this appears before the property methods,